pkg/kubeapiserver/options/authentication.go

```diff
@@ -83,6 +100,9 @@ type OIDCAuthenticationOptions struct {
 	GroupsPrefix   string
 	SigningAlgs    []string
 	RequiredClaims map[string]string
+
+	// areFlagsConfigured is a function that returns true if any of the oidc-* flags are configured.
+	areFlagsConfigured func() bool
```

Comment:
Some of the unit tests are failing because they are trying to diff this private field.